src/imio/smartweb/common/utils.py

```
@@ -20,11 +20,35 @@
+
+def get_json(url, auth=None, timeout=5):
+    language = api.portal.get_current_language()
+    headers = {"Accept": "application/json", "Cookie": f"I18N_LANGUAGE={language}"}
+    if auth is not None:
+        headers["Authorization"] = auth
+    try:
+        response = requests.get(url, headers=headers, timeout=timeout)
+    except requests.exceptions.Timeout:
+        logger.warning(f"Timeout raised for requests : {url}")
+        return None
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)